spring-security-modules/spring-security-core-2/src/main/java/com/baeldung/exceptionhandler/security/SecurityConfig.java

```diff
@@ -41,30 +43,23 @@ public UserDetailsService userDetailsService() {
 
     @Bean
     public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
-        http.csrf()
-            .disable()
-            .httpBasic()
-            .disable()
-            .authorizeRequests()
-            .antMatchers("/login")
+        http.csrf(AbstractHttpConfigurer::disable)
+            .httpBasic(AbstractHttpConfigurer::disable)
+            .authorizeHttpRequests(auth -> auth
+            .requestMatchers("/login")
             .permitAll()
-            .antMatchers("/customError")
+            .requestMatchers("/customError")
             .permitAll()
-            .antMatchers("/access-denied")
+            .requestMatchers("/access-denied")
             .permitAll()
-            .antMatchers("/secured")
+            .requestMatchers("/secured")
             .hasRole("ADMIN")
             .anyRequest()
-            .authenticated()
-            .and()
-            .formLogin()
-            .failureHandler(authenticationFailureHandler())
-            .successHandler(authenticationSuccessHandler())
-            .and()
-            .exceptionHandling()
-            .accessDeniedHandler(accessDeniedHandler())
-            .and()
-            .logout();
+            .authenticated())
+            .formLogin(form -> form.failureHandler(authenticationFailureHandler())
+                    .successHandler(authenticationSuccessHandler()))
```

Comment:
Can you align the indention to be the same 4spaces please, I believe that is how the baeldung formatting should be. Thanks!